hkmc2/shared/src/test/mlscript/ucs/syntax/Else.mls

```diff
@@ -98,3 +93,31 @@ if audits.has(1) === true do
 else
   1
 //│ = 1
+
+
+(if true then
+    5
+  else 1
+) + 3
+//│ = 8
+
+:fixme
+(if true
+  then
+    5
+  else 1
+) + 3
+//│ ╔══[PARSE ERROR] Unexpected 'else' keyword here
+//│ ║  l.108: 	  else 1
+//│ ╙──       	  ^^^^
+//│ = 8
+
+:fixme
```

Comment:
I'm not so sure we want to support that... Anyway, this will be addressed in the parser rewrite (no need to change it).